src/utils/time/calculateTimeLeft.ts

```diff
@@ -1,22 +1,11 @@
-export const calculateTimeLeft = (
-  createdAt: number,
-  differenceVsBlockChainTime: number,
-  actionPeriod?: number,
-) => {
-  const difference = actionPeriod
-    ? createdAt +
-      actionPeriod * 1000 -
-      (Date.now() + differenceVsBlockChainTime)
-    : 0;
-
-  if (difference > 0) {
+export const splitTimePeriod = (period: number) => {
```

Comment:
Maybe the file should be rename to `splitTimeLeft` as well for consistency?

Reply:
Sure :+1: